spec/Cabin/Infotainment.vspec

```diff
@@ -189,6 +189,21 @@ HMI.DayNightMode:
   allowed: ['DAY', 'NIGHT']
   description: Current display theme
 
+HMI.IsScreenAlwaysOn:
+  datatype: boolean
+  type: actuator
+  description: used to prevent the screen going black if no action placed
```

Comment:
I think description is ok, but please change case to `Used...` and add a dot/period at end.

Reply:
I have updated the code.